### ISSUE TITLE
Abnormal hip extension

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5909]
**Preferred term label:** Abnormal hip extension

**Synonyms**

**Definition (free text, please give PubMed ID)** Limited extension or hyperextension of the hip.

**Parent term (use hpo.jax.org/app)** Abnormality of the hip joint HP:0001384

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
This could be the parent term for: Limited hip extension HP:0003093; Hip joint hypermobility HP:0045087

**Comment by pnrobinson** [comment]
We do not at present have parent terms for limited mobility and hypermobility. I would suggest let's leave this now, we are hoping to have a skeletal session for the HPO in the near-ish future.
